runelite-client/src/main/java/net/runelite/client/plugins/statusbars/StatusBarsOverlay.java

```diff
@@ -175,9 +175,13 @@ else if (poisonState >= 1000000)
 			final MenuEntry entry = menu[menuSize - 1];
 			final Effect change = itemStatService.getItemStatChanges(entry.getIdentifier());
 
-			if (change != null & entry.getParam1() == WidgetInfo.INVENTORY.getId())
+			if (change != null &
```

Comment:
I dont think this is really the correct way to go about this... maybe check for item id?

Reply:
this should be fine now, it now checks for the id.

Reply:
Do all spicy stews share the same ID?

Reply:
Also, isn't that ID already stored in ItemID? Use that instead.

Reply:
only the yellow stew throws that exception and yes its the same ID for all amount of spices,

Reply:
Ah, right. You should still be using the ItemID definition rather than adding it as a constant.